functions/tab.terminator.fish

```diff
@@ -0,0 +1,3 @@
+function tab.terminator -a cdto cmd
+  terminator --new-tab --working-directory "$cdto" --execute "$cmd"
```

Comment:
i don't use terminator… can you verify for me that this keeps an interactive session open after `$cmd` is executed?

Reply:
Just tested this, terminator doesn't keep the session open since all we've done is issue it a single command.